src/Model/DocsAsHtml.hs

```diff
@@ -241,15 +240,6 @@ splitOnPathSep str
   | '\\' `elem` str = splitOn "\\" str
   | otherwise       = [str]
 
-renderFixity :: P.Fixity -> Html ()
-renderFixity (P.Fixity associativity precedence) =
```

Comment:
Because it's easier to understand? I don't really like the `<code>` part of the rendering of fixity declarations actually, if we're getting rid of one of them I'd rather it were that one.